### DISCUSSION TITLE
More information on working.

### DISCUSSION THREAD

**Discussion by Bugaddr** [textbrowser/smoke#20] [Q&A] [answered]
Hey all, i got to know about this app on fdroid. But due to limited info on fdroid about working of this app, i got confused . But i really wanna try this app. Small info regarding this is much appreciated.
Ps: sorry for bad english
Thanks

**Comment by textbrowser** [comment]
There is an official document. There is information in the README.

**Reply by Manamama** [reply]
I have been using computers since mid-1980s: I started with coding directly in assembler back then, yet I am not a coder by far. 
I have read the Readme. I have read the blurb in F-Droid quoted below. I have read stuff here, on mobile. 
I still have no idea what it does. It reminds me of the Silicon Valley film and the head geek - do rewatch. This is your "explanation": 

Summary of Smoke

* Aliases. Preserve your contacts.
* Almost zero-dependency software.
* Application lock.
* Argon2id and PBKDF2 key-derivation functions.
* Automatic, oscillatory public-key exchange protocol, via SipHash.
* BSD 3-clause license.
* Content is recorded via authenticated encryption.
...
It shows what it is (SipHashes and Argon somethings) and not "where is the beef?", "what is there in for ME" - the user.

**Reply by textbrowser** [reply]
That's fine.

**Comment by textbrowser** [comment, marked as answer]
Also this discussion: https://github.com/textbrowser/smoke/discussions/13.
